lib/huginn_acumen_product_agent/concerns/acumen_product_query_concern.rb

```diff
@@ -110,6 +110,7 @@ def parse_product_request(products)
             variant['@type'] = 'ProductModel'
             variant['isDefault'] = field_value(p, 'Inv_Product.OnWeb_LinkOnly') == '0'
             variant['isTaxable'] = field_value(p, 'Inv_Product.Taxable') == '1'
+            variant['is_master'] = field_value(p, 'Inv_Product.OnWeb_LinkOnly') == '0'
```

Comment:
A `ProductModel` is a `Product`, so to keep things consistent, it should be `variant['acumenAttributes']['is_master']`. I'm also not sure why it's necessary, since the `sku` and `identifier` will match on its master variant.

Reply:
@Ken-Michalak `is_master` is not a variant identifier.

Every product has a master variant (driven by `isDefault`, but every _title_ has a master _product_. In most cases, the master product is the physical product, but for some digital only titles, the master product would be the digital one.

ultimately, `is_master` controls which products appear on search results/category pages.

Reply:
@Ken-Michalak @mrogers-5s  I changed this a bit so that we can get the is_master to show up on the variant rather than just the wrapper product as true. Let me know what you think

https://trello.com/c/yolBI8jk/134-asbat-see-products-syncd-correctly-connecting-ismaster-to-custom-field-in-bc